CS483/CS483/Kartaclysm/StateMachine/Gameplay/StateRacing.h

```diff
@@ -72,12 +72,18 @@ namespace Kartaclysm
 		bool m_bSuspended;
 
 	private:
+		struct RacerResults
+		{
+			std::string m_strRacerId;
+			float m_fRaceTime;
+		};
+		std::vector<RacerResults> m_vRaceResults;
```

Comment:
Good point, forgot that you also need the position implicitly stored by a vector.